R/tbl_propdiff.R

```diff
@@ -0,0 +1,365 @@
+#' Calculating unadjusted and adjusted differences in rates
+#'
+#' This function calculates the unadjusted or adjusted difference in rates with confidence interval.
+#'
+#' @param data A data frame
+#' @param outcome A vector of outcome variable names. Outcome variables can be numeric, character or factor, but must have two and only two non-missing levels
+#' @param predictor The name of the predictor (stratifying) variable. This can be numeric, character or factor, but must have two and only two non-missing levels
+#' @param covariates A vector of variable names that the difference in rates will be adjusted for. The p-value displayed will be from a multivariable logistic regression model containing these covariates. The default option is `NULL` which calculates an unadjusted difference in rates.
+#' @param method The method for calculating confidence intervals around the adjusted difference in rates. The 'centile' option is the default and uses 2000 bootstrap resamples by default and calculates the confidence interval using centiles (by default, 2.5th and 97.5th centiles for a 95% confidence interval). The 'mean' option can also be selected, which uses a default 200 bootstrap resamples and uses the standard deviation for the mean adjusted difference across all resamples to calculate the confidence interval.
+#' @param conflevel Confidence level of the returned confidence interval. Must be a single number between 0 and 1. The default is a 95% confidence interval.
```

Comment:
Can you check this arg name against tbl_regression()?  I think it should be `conf.level=`

Reply:
changed this to conf.level